src/components/block/ProgramCard/ProgramCard.module.css.d.ts

```diff
@@ -0,0 +1,6 @@
+export const programCardContainer: string;
```

Comment:
Are we supposed to commit all these files to version control? I feel like this is probably not a good idea because it's just redundant bookkeeping whenever we make a change to the actual CSS files. Do you know how these .d.ts files generated from CSS modules are meant to be used?

Reply:
I've never seen anything like this before. This feels off, but I'm glad I'm presenting it here for further discussion.